### PR TITLE
[1491] Add deserializable_resource call to AccessRequestsController

### PR DESCRIPTION
**Pull request by tvararu**
This is necessary to correctly interpret the parameters sent in by the support app.

Paired with @davidgisbey.

### Checklist

- [x] Make sure all information from the Trello card is in here
- [x] Attach to Trello card
- [x] Rebased master
- [x] Cleaned commit history
- [x] Tested by running locally

### CODE REVIEW

**Comment by tvararu** [comment]
[Switch manual access_request creation api calls to new backend endpoint](https://trello.com/c/ksqmnJYW/1491-switch-manual-accessrequest-creation-api-calls-to-new-backend-endpoint)

File: app/controllers/api/v2/access_requests_controller.rb [hunk 1]
```diff
@@ -1,6 +1,7 @@
 module API
   module V2
     class AccessRequestsController < API::V2::ApplicationController
+      deserializable_resource :access_request, only: %i[create]
```
Comment (dankmitchell) [inline app/controllers/api/v2/access_requests_controller.rb]:
```suggestion
      deserializable_resource :access_request, only: :create
```

Comment (tvararu) [reply app/controllers/api/v2/access_requests_controller.rb]: Ehhh if we add another one then Rubocop will be like "use `%i[`, it's fine.

Comment (dankmitchell) [reply app/controllers/api/v2/access_requests_controller.rb]: Yep. Ship it

**Pull request merged**